etc/sync-function-template.js

```diff
@@ -99,31 +99,39 @@ function(doc, oldDoc) {
   function validateDoc(doc, oldDoc, docDefinition, docType) {
     var validationErrors = [ ];
 
-    if (!(docDefinition.allowAttachments) && doc._attachments) {
```

Comment:
I recommend viewing this file's changes with `?w=1` to ignore whitespace changes.